### PR TITLE
Models/Migrations

### PR DESCRIPTION
**Pull request by jroodman**
This PR includes the set of migrations, models, and tests for the GroundRules application.

### CODE REVIEW

**Review by nhunzaker — APPROVED** [review]
Very neat! I'm not a Rails dev, so I can't really pass off on this, but I thought everything was really clear.

File: spec/models/device_problem_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  it { should belong_to(:device) }
+
+  let!(:device) do
+    Device.create(name: 'Fake Device Name', image_url: 'http://www.fakeimageurl.com/')
+  end
+
+  subject do
+    DeviceProblem.create(
+      problem: 'Fake Problem',
+      content: JSON.parse("{ \"fake_content\": \"content\" }"),
```
Comment (rysospicy) [inline spec/models/device_problem_spec.rb]: You can pass a Ruby has into a JSON-type column and it'll handle it correctly.  You don't have to do the extra work of building up a valid JSON string and then parsing it.

File: app/models/device_problem.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class DeviceProblem < ApplicationRecord
+
+  belongs_to :device
+
+  validates :problem, :content, :device_id, presence: true
+
+  def to_s
+    problem
```
Comment (rysospicy) [inline app/models/device_problem.rb]: I don't know if this name is super clear as you'll be calling something like `device_problem.problem`.  Would something like `device_problem.statement` or `device_problem.headline` make any more sense?

**Review by rysospicy — APPROVED** [review]
Two small comments.  Lookin' good! 👍

File: spec/models/device_problem_spec.rb (hunk 1 above)
Comment (jroodman) [reply spec/models/device_problem_spec.rb]: Should I pass an object in here then or the string?

File: app/models/device_problem.rb (hunk 2 above)
Comment (jroodman) [reply app/models/device_problem.rb]: I like statement, I'll swap that out. Thanks!

**Pull request merged**